packages/web3-common/src/errors.ts

```diff
@@ -409,3 +410,17 @@ export class InvalidPrivateKeyError extends Web3Error {
 		super(`Invalid value given "${String(value)}". Error: not a valid string or buffer.`);
 	}
 }
+
+export class InvalidKdfError extends Web3Error {
+	public code = ERR_UNSUPPORTED_KDF;
+	public constructor(){
+		super(`Error: invalid key derivation function`)
+	}
+}
+
+export class KeyDerivationError extends Web3Error {
+	public code = ERR_UNSUPPORTED_KDF;
+	public constructor() {
+		super(`Error: Key derivation failed - possibly wrong password`)
```

Comment:
```suggestion
		super(`Invalid key derivation function`)
	}
}

export class KeyDerivationError extends Web3Error {
	public code = ERR_UNSUPPORTED_KDF;
	public constructor() {
		super(`Key derivation failed - possibly wrong password`)
```

No need to include `Error:` prefix, no other error message has it and JavaScript will already include it in native Error message